plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/PullParams.java

```diff
@@ -96,19 +124,49 @@ public String getRegistry() {
         return registry;
     }
 
+    public String getNamespace() {
+        return namespace;
+    }
+
+    public AuthConfigs getAuthConfigs() {
+        return authConfigs;
+    }
+
+    /**
+     * Returns FQN of image for pull.
+     * It has following format: [registry/][namespace/]image
+     */
+    public String getImageFqn() {
```

Comment:
Are you sure that you need that method? Why docker connector doesn't concat strings instead?